src/ForkCMS/Bundle/InstallerBundle/Form/Type/LanguagesType.php

```diff
@@ -69,34 +71,34 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         ;
     }
 
-    public function setDefaultOptions(OptionsResolverInterface $resolver): void
+    public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
             'data_class' => 'ForkCMS\Bundle\InstallerBundle\Entity\InstallationData',
         ]);
     }
 
-    public function getName(): string
+    public function getBlockPrefix(): string
     {
         return 'install_languages';
     }
 
     protected function getInstallableLanguages(): array
     {
         return [
-            'en' => 'English',
-            'zh' => 'Chinese',
-            'nl' => 'Dutch',
-            'fr' => 'French',
-            'de' => 'German',
-            'el' => 'Greek',
-            'hu' => 'Hungarian',
-            'it' => 'Italian',
-            'lt' => 'Lithuanian',
-            'ru' => 'Russian',
-            'es' => 'Spanish',
-            'sv' => 'Swedish',
-            'uk' => 'Ukrainian',
+            'English' => 'en',
```

Comment:
at this moment it is hardcoded everywhere in fork, I changed it around because it needs to be like this in symfony3, labels are now the key and values the value